orders/views.py

```diff
@@ -1,3 +1,65 @@
-from django.shortcuts import render
+import json
+
+from django.http.response  import JsonResponse
+from django.views          import View
+
+from .models               import Cart
+
+class CartView(View):
+    def get(self, request):
+            carts  = Cart.objects.filter(user_id = request.user_id)
```

Comment:
테스트를 해보셨을까요? 코드에 오류가 있어보입니다
```suggestion
            carts  = Cart.objects.filter(user_id = request.user.id)
```